content/articles/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/index.md

```diff
@@ -0,0 +1,220 @@
+```MATLAB
+f_sub = subs(f, w, [-pi: pi])
+```
+`subs` mean symbolic substitution. It means all the symbolic variables are substituted. This function replaces the symbolic variable in `f` with the values of `w`, which is in the range of `-pi` to `pi`. When we do this, we get the values below:
+```MATLAB
+f_sub =
+ 
+[ -2/(- 1 + pi*2i), 39614081257132168796771975168/383180597959374460738291811785 + 169674450797966737934326431744i/383180597959374460738291811785, 39614081257132168796771975168/123059858877705322463182898633 + 90446288283702400340782481408i/123059858877705322463182898633, 39614081257132168796771975168/21395444824564859375161886153 + 11218125769438062747238531072i/21395444824564859375161886153, 39614081257132168796771975168/78187355799953071474228774345 - 68010036744826274846305419264i/78187355799953071474228774345, 39614081257132168796771975168/293435591803869958760383563209 - 147238199259090612439849369600i/293435591803869958760383563209, 39614081257132168796771975168/667140152836315521233626252745 - 226466361773354950033393319936i/667140152836315521233626252745]
+```
+Now, we plot these values using the `plot()` function.
+```MATLAB
+subplot(2, 1, 2);
+plot(angle(double(f_sub)));   %angle are the w values.
+title('phase spectrum')   % title
+```
+`plot(angle(double(data_value)))` gives the phase spectrum plot. The function `subplot()` is for making a subplot. The `title()` gives the plot a title.  
+
+![phase spectrum](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-three.png)
+
+*Phase spectrum plot*
+
+Now lets plot the magnitude response using the same values of `w`.
+```MATLB
+subplot(2, 1, 1); 
+plot(abs(double(f_sub))); %magnitude spectrum plot
+title('magnitude spectrum')
+```
+The code `plot(abs(double(data_values)))` gives the magnitude spectrum. This plot uses the absolute values of the data, thus `abs()`.
+
+![magnitude spectrum](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-four.png)
+
+*Magnitude spectrum plot*
+
+### Example 2
+lets look at another example:
+![example 2](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-nine.jpeg)
+
+We will have:
+```MATLAB
+syms t w T W
+f = int(1*exp(-j*w*t), t, [-T, T])
+```
+The output here is:
+```MATLAB
+f =
+ 
+(2*sin(T*w))/w
+``` 
+pretty(f) gives:
+```MATLAB
+2 sin(T w)
+----------
+     w
+```
+Now, we find the values of `w` and then plot these values. To find the values of `w`, we use the `subs()` function.
+```MATLAB
+f_sub = subs(f, T, 0.5)
+ 
+f_sub =
+ 
+(2*sin(w/2))/w     % the output after execution of the code above.
+```
+We are now plotting the absolute values of the variable `f-sub`.
+```MATLAB
+ezplot(abs(f_sub), [-pi: pi])
+ezplot(abs(f_sub), [-2*pi: 2*pi])
+ezplot(abs(f_sub), [-4*pi: 4*pi])
+ezplot(abs(f_sub), [-8*pi: 8*pi])
+```
```

Comment:
For this part, you can seperate and put them in different snippets(blocks). Thereafter, add each output image below the corresponding snippets. This is because when I ran this, it only showed the plot for the last line.